server/static/bindata_assetfs.go

```diff
@@ -198,8 +198,8 @@ func staticJsJquery321MinJs() (*asset, error) {
 // It returns an error if the asset could not be found or
 // could not be loaded.
 func Asset(name string) ([]byte, error) {
-	cannonicalName := strings.Replace(name, "\\", "/", -1)
```

Comment:
this file is generated, it should not be changed

Reply:
For my reference, this appears to now live in https://github.com/go-bindata/go-bindata -- I'll add it to my todo list.

Reply:
go-bindata/go-bindata#47